test/hotspot/jtreg/compiler/c2/irTests/TestIRFma.java

```diff
@@ -85,99 +86,99 @@ public void assertResult(double a, double b, double c) {
     }
 
     @Test
-    @IR(counts = {IRNode.FMSUB, "> 0"},
-        applyIfCPUFeature = {"asimd", "true"})
+    @IR(counts = {IRNode.FMSUB_F, "> 0", IRNode.NEG_F, "> 0"},
```

Comment:
What is the reason for adding the "IRNode.NEG_F" ?

Reply:
I think I need to reconsider the test improvement, will update the pr later.

Thank for having a look.

Reply:
I'll keep the test simple, let IR verification to verify the optimal FmaF/D nodes are generated, and let asserts to verify the functionality of these nodes are right. So it's a bit different from another platform, which verify the specific instructs are finally generated in "FINAL CODE" phase, which is not bad, but seems over-complicated and over-white-boxed, and different platforms could have different instruct names for the same IR node (which is the case in FMA on riscv and aarch64). And I'll leave aarch64 untouched.